registry/app.go

```diff
@@ -88,6 +88,8 @@ func NewApp(configuration configuration.Configuration) *App {
 }
 
 func (app *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Set a header with the Docker Distribution API Version for all responses.
+	w.Header().Set("Docker-Distribution-API-Version", "2.0")
```

Comment:
If you're adding headers, would it be useful to have a release/build version somewhere in a header? Thinking of support questions and being able to check what *exact* version a user was using.

(Sorry if this is too much noise, idea came up reading this) :)